### PR TITLE
css introduced to monitorix file only

### PR DESCRIPTION
**Pull request by z-eos**

### CODE REVIEW

File: monitorix [hunk 1]
```diff
@@ -24,7 +24,7 @@ require 5.006;
 use strict;
 use warnings;
 use FindBin qw($Bin);
-use lib $Bin . "/lib", "/usr/lib/monitorix";
+use lib $Bin . "/lib", "/usr/local/share/monitorix";
```
Comment (mikaku) [inline monitorix]: You don't need to change that path.
Revert it, please.

File: monitorix [hunk 2]
```diff
@@ -223,51 +223,48 @@ EOF
 	if(!open(OUT, "> $config{base_dir}/index.html")) {
 		die "unable to create '${config{base_dir}}index.html': $!";
 	}
+	my $css = sprintf("%scss/monitorix-%s.css", $base_url, $config{theme_color});
```
Comment (mikaku) [inline monitorix]: The names of the themes don't need to be prefixed as `monitorix-` it's redundant.

Comment (z-eos) [reply monitorix]: fixed

**Comment by mikaku** [comment]
Thanks.

**Pull request merged**